### PR TITLE
System healthcheck

### PR DESCRIPTION
**Pull request by Nesze**

### CODE REVIEW

File: service-files/system-healthcheck.service [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Requires=docker.socket
+After=docker.socket
+
+[Service]
+Environment="DOCKER_APP_VERSION=latest"
+Environment="PROC_DIR=host_proc"
+ExecStartPre=-/usr/bin/docker kill %p-%H
+ExecStartPre=-/usr/bin/docker rm %p-%H
+ExecStartPre=/usr/bin/docker pull up-registry.ft.com/coco/system-healthcheck:$DOCKER_APP_VERSION
+ExecStart=/bin/bash -c "/usr/bin/docker run --rm --name=%p-%H -e PROC_DIR=$PROC_DIR -v /:/$PROC_DIR:ro up-registry.ft.com/coco/system-healthcheck:$DOCKER_APP_VERSION"
```
Comment (Zapadlo) [inline service-files/system-healthcheck.service]: You don't need to have %H in the container name, and considering how picky it is about characters, I think its actively better not to.

**Comment by catalin-ilea** [comment]
lgtm

File: service-files/system-healthcheck.service [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Requires=docker.socket
+After=docker.socket
+
+[Service]
+Environment="DOCKER_APP_VERSION=latest"
+Environment="PROC_DIR=host_proc"
+ExecStartPre=-/usr/bin/docker kill %p
+ExecStartPre=-/usr/bin/docker rm %p
+ExecStartPre=/usr/bin/docker pull up-registry.ft.com/coco/system-healthcheck:$DOCKER_APP_VERSION
+ExecStart=/bin/bash -c "/usr/bin/docker run --rm --name=%p -e PROC_DIR=$PROC_DIR -v /:/$PROC_DIR:ro up-registry.ft.com/coco/system-healthcheck:$DOCKER_APP_VERSION"
```
Comment (Zapadlo) [inline service-files/system-healthcheck.service]: You are mounting ALL of host filesystem into the container, probably overkill and should only mount /proc:

``.. -v /proc:/$PROC_DIR:ro

Comment (Nesze) [inline service-files/system-healthcheck.service]: I did that on purpose, I thought for disk space check, I have to mount the whole fs.

Comment (Zapadlo) [inline service-files/system-healthcheck.service]: Right, makes sense.

**Comment by Zapadlo** [comment]
+1

**Pull request merged**